File: s3-inf-ppo/laboratoria/lab02/lab02b.java
import java.util.ArrayList;
import java.util.Random;

class Student {

	public String studentNo;

	public void setStudentNo(String studentNo) {
		this.studentNo = studentNo;
	}
		
	public String getStudentNo() {
		return this.studentNo;
	}

}

public class lab02b {

	static int students_count = 10;

	public static void main(String[] args) {
		ArrayList<Student> students = new ArrayList<Student>();
		
		for(int i = 0; i < students_count; i++) {
			Student student = new Student();
			student.setStudentNo(getRandomStudentNumber());
			students.add(student);
		}
		
		System.out.println("Students group have been filled.");
		
		for(int i = 0; i < students.size(); i++) {
			Student student = students.get(i);
			System.out.println(student.getStudentNo());
		}
	}

	protected static String getRandomStudentNumber() {
		Random rand = new Random();
		return String.valueOf(rand.nextInt(2000) + 38000);
	}

}

File: s3-inf-ppo/laboratoria/lab02/lab02b.cs
using System;
using System.Collections.Generic;

public class Student
{
    public string studentNo;

    public void SetStudentNo(string studentNo)
    {
        this.studentNo = studentNo;
    }

    public string GetStudentNo()
    {
        return this.studentNo;
    }
}

public class Lab02b
{
    const int students_count = 10;
    static Random rand = new Random();

    public static void Main(string[] args)
    {
        List<Student> students = new List<Student>();

        for (int i = 0; i < students_count; i++)
        {
            Student student = new Student();
            student.SetStudentNo(GetRandomStudentNumber());
            students.Add(student);
        }

        Console.WriteLine("Students group have been filled.");

        for (int i = 0; i < students.Count; i++)
        {
            Student student = students[i];
            Console.WriteLine(student.GetStudentNo());
        }
        Console.ReadKey();
    }

    protected static string GetRandomStudentNumber()
    {
        return (rand.Next(2000) + 38000).ToString();
    }
}